nextstrain_profiles/nextstrain-gisaid/builds.yaml

```diff
@@ -273,30 +273,18 @@ subsampling:
 
   # Custom subsampling logic for region Asia over 1m
   # Grouping by division
-  # Separating three buckets for China, India and elsewhere
+  # Grouping by country weighted by population size
   # 4375 total
   # 4:1 ratio of recent to early
   # 4:1 ratio of focal to context
-  # 3:2:2 proportions of Asia, China, India
   nextstrain_region_asia_grouped_by_division_1m:
     # Early focal samples for Asia
     asia_early:
-      group_by: "division year month"
-      max_sequences: 300
-      max_date: "--max-date 1M"
-      exclude: "--exclude-where 'region!=Asia' 'country=China' 'country=India'"
-    # Early focal samples for China
-    china_early:
-      group_by: "division year month"
-      max_sequences: 200
-      max_date: "--max-date 1M"
-      exclude: "--exclude-where 'country!=China'"
-    # Early focal samples for India
-    india_early:
-      group_by: "division year month"
-      max_sequences: 200
+      group_by: "country year month"
+      group_by_weights: "defaults/population_weights.tsv"
```

Comment:
Implementation tweak is looking good and I think the current preview results are what it's going to be.

The next step here is to figure out how to define the weights file. Currently it's saved as `defaults/population_weights.tsv`, which I think is the best place to put the file. However, unlike other files in `defaults`, this isn't used by a single config key and practically not meant to be overridden (to do so would require overriding all 7 references to it in the subsampling YAML).

No better ideas are coming to mind, so suggestions are welcome.